test/unittest/layers/layers_golden_tests.cpp

```diff
@@ -209,11 +224,18 @@ static void compareRunContext(RunLayerContext &rc, std::ifstream &file,
       for (unsigned int idx = 0; idx < total; idx++) {
         auto d1 = t1.getValue<_FP16>(idx);
         auto d2 = t2.getValue<_FP16>(idx);
-        auto float_eq = [](_FP16 a, _FP16 b) {
-          constexpr auto eps = 1e-2;
-          if (a < b)
-            std::swap(a, b);
-          return (a - b) < eps;
+        auto float_eq = [skip_cos_sim](_FP16 a, _FP16 b) {
+          if (skip_cos_sim) {
+            constexpr auto eps = 1e-1;
+            if (a < b)
+              std::swap(a, b);
+            return (a - b) < eps;
+          } else {
+            constexpr auto eps = 1e-2;
+            if (a < b)
+              std::swap(a, b);
+            return (a - b) < eps;
+          }
```

Comment:
minimize the code redundancy?

```suggestion
          constexpr auto eps =  (skip_cos_sim) ? 1e-1 : 1e-2;
          if (a < b)
            std::swap(a, b);
          return (a - b) < eps;
```

Reply:
I also tried it, but it throws:
```
../test/unittest/layers/layers_golden_tests.cpp:228:33: error: lambda capture of ‘skip_cos_sim’ is not a constant expression
  228 |           constexpr auto eps = (skip_cos_sim) ? 1e-1 : 1e-2;
      |                                 ^~~~~~~~~~~~
```

Reply:
That block is unavoidable for near-zero Tensor case... (though I don't like it either) Any better idea?